implementations/elixir/ockam/ockam_services/lib/services.ex

```diff
@@ -38,7 +38,7 @@ defmodule Ockam.Services do
           Ockam.Services.Provider
         ]
 
-    Supervisor.start_link(children, strategy: :one_for_one, name: __MODULE__)
+    Supervisor.start_link(children, strategy: :one_for_all, intensity: 0, name: __MODULE__)
```

Comment:
what the reason to switch to :one_for_all here?   we want to restart services if a transport crash?

Reply:
This is a top-level supervisor, `Ockam.Services.Provider` is a services supervisor.
If this one supports restarts, then too many restarts in services will cause `Ockam.Services.Provider` to restart and that will get rid of all services which were started not from config, e.g. api gateways which are started from application start up.